packages/blocks/src/api/utils.js

```diff
@@ -237,11 +237,16 @@ export function getAccessibleBlockLabel(
  * Ensure attributes contains only values defined by block type, and merge
  * default values for missing attributes.
  *
- * @param {string} name       The block's name.
- * @param {Object} attributes The block's attributes.
+ * @param {string}  name                     The block's name.
+ * @param {Object}  attributes               The block's attributes.
+ * @param {boolean} sanitizeUniqueAttributes Whether to sanitize unique attributes.
  * @return {Object} The sanitized attributes.
  */
-export function __experimentalSanitizeBlockAttributes( name, attributes ) {
+export function __experimentalSanitizeBlockAttributes(
+	name,
+	attributes,
+	sanitizeUniqueAttributes = false
```

Comment:
Agreed -- the named option is a big improvement. I initially thought to create another function for this (since as @glendaviesnz pointed out, de-duping is also a bit out of place as part of 'sanitize' functionality), but it's a bit rough on code duplication. I can take another stab at refactoring here if we go with this approach.